k8s/hyperconverged/tutorial-configure-openebs-aks.md

```diff
@@ -0,0 +1,264 @@
+
+```
+devops@Azure:~$ az group create -l eastus -n aks-openebs-cluster-rg
+
+{
+  "id": "/subscriptions/c73e5d9b-160d-45e7-b84e-f243928ed8ed/resourceGroups/aks-openebs-cluster-rg",
+  "location": "eastus",
+  "managedBy": null,
+  "name": "aks-openebs-cluster-rg",
+  "properties": {
+    "provisioningState": "Succeeded"
+  },
+  "tags": null
+}
+```
+**Note:** The example commands below were run on a Kubernetes cluster aks-openebs-cluster  in zone eastus with resource group aks-openebs-cluster-rg. When you copy paste the command, ensure that you use the details from your project.
+
+#### Create  cluster
+
+Use the _az aks create_ command to create an AKS cluster. The following example creates a cluster named *aks-openebs-cluster* with two nodes and the vm size is Standard_A2_V2. For more details, click [node vm size](https://docs.microsoft.com/en-us/azure/cloud-services/cloud-services-sizes-specs) 
+
+```
+devops@Azure:~$ az aks create -n aks-openebs-cluster -g aks-openebs-cluster-rg --node-count 2 --kubernetes-version 1.8.7 --node-vm-size Standard_A2_v2 --generate-ssh-key
+```
+After few minutes, the cluster with the above configuration will be created successfully.
+
+#### Connect to the cluster
+
+To manage a Kubernetes cluster, use _kubectl_, the Kubernetes command-line client. If you're using Azure Cloud Shell, kubectl is already installed. 
+
+To configure kubectl to connect to your Kubernetes cluster, use the _az aks get-credentials_ command. This step downloads credentials and configures the Kubernetes CLI to use them.
+
+```
+devops@Azure:~$ az aks get-credentials -g aks-openebs-cluster-rg -n aks-openebs-cluster
+
+Merged "aks-openebs-cluster" as current context in /home/cloudbyte/.kube/config
+```
+
+To verify the connection to your cluster, use the _kubectl get_ command to return a list of the cluster nodes. Note that this can take a few minutes to appear.
+
+```
+devops@Azure:~$ kubectl get nodes
+NAME                       STATUS    ROLES     AGE       VERSION
+aks-nodepool1-46849391-0   Ready     agent     20h       v1.8.7
+aks-nodepool1-46849391-1   Ready     agent     20h       v1.8.7
+```
+
+## iSCSI Configuration
+
+Create Public IP addresses and associate them to all the nodes' network interfaces to configure iSCSI initiator. You can create and associate the Public IP address to the nodes by using Azure portal or Azure CLI
+
+**Create Public IP address using Azure Portal**
+- Go to the Azure Portal select **All Services**  search **Public IP addresses** and click it. 
+- Click **Create Public IP adresses** and provide the required details such as **Name** , select _Static_ in the **IP address Assignment **, **Resource Group** and the **location** where the selected resource group resides. Click **Create**. You can either select the existing resource group or create a new resource group. Select the resource group that MC_aks-openebs-cluster-rg_aks-openebs-cluster_eastus.
```

Comment:
select an existing...
Select the MC_aks-openebs-cluster-rg_aks-openebs-cluster_eastus resource group.